CHANGELOG.md

```diff
@@ -17,7 +17,9 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased](https://github.com/o1-labs/snarkyjs/compare/3fbd9678e...HEAD)
 
-> no unreleased changes yet
+### Added
+
+- **Foreign field arithmetic** exposed through the `createForeignField()` class factory https://github.com/o1-labs/snarkyjs/pull/985
```

Comment:
Nit: Do you think it's helpful to have a more descriptive entry in the changelog for such an awesome feature? :D 

Something like:
`
Introduced 'ForeignField', a class for prime order finite fields distinct from the native Field. Supports arithmetic operations, helper methods, and allows optional bypass of automatic validity checks for optimized code. Supports prime moduli up to 259 bits. https://github.com/o1-labs/snarkyjs/pull/985
`

Reply:
I think you want to be more restrictive than just checking less than 259 bits, but also check it's less than the max modulus specifically, which is commented as `926336713898529563388567880069503262826888842373627227613104999999999999999607` here: https://github.com/MinaProtocol/mina/blob/cf5664de9cdd8459edc87cc4c9759b3cbe534cf3/src/lib/crypto/kimchi_backend/gadgets/foreign_field.ml#L42